Content.Server/Body/Systems/MetabolizerSystem.cs

```diff
@@ -185,4 +197,13 @@ private void TryMetabolize(EntityUid uid, MetabolizerComponent? meta=null, Mecha
             }
         }
     }
+    public sealed class ApplyMetabolicMultiplierEvent : EntityEventArgs
+    {
+        // The entity whose metabolism is being modified
+        public  EntityUid Uid;
+        // What the metabolism's update rate will be multiplied by
+        public  float Multiplier;
+        // Apply this multiplier or ignore / reset it?
+        public bool Apply;
+    }
```

Comment:
this should probably be in `BedSystem` since its more relevant there

Reply:
I could see other sources of this effect in the future, like chems

Reply:
Also I ended up moving the systems around in the first place because of friend-protected classes IIRC

Reply:
aight